src/crystal/init_project/template/gitignore.ecr

```diff
@@ -0,0 +1,4 @@
+/.deps/
+/.deps.lock
```

Comment:
So the question is, what we want here as a default? Library or Application behavior of git ignore?